appcli/models/configuration.py

```diff
@@ -25,7 +25,7 @@ class Hooks(NamedTuple):
      function is used."""
     is_valid_variables: Callable[
         [click.Context, Dict, Dict], bool
-    ] = lambda w, x, y: is_matching_dict_structure(x, y)
+    ] = lambda w, x, y: True
```

Comment:
A comment here explaining why this is true would be useful. It isn't obvious that `is_matching_dict_structure` should be the call unless you go hunting further down the file. Something like
`TODO: (Github issue #77) Instead of True the call should be is_matching_dict_structure(x, y) which is currently disabled`